docs/source/about_dataset_load.rst

```diff
@@ -0,0 +1,101 @@
+Build and load
+==============
+
+Nearly every deep learning workflow begins with loading a dataset, which makes it one of the most important steps. With 🤗 Datasets, there are more than 900 datasets available to help you get started with your NLP task. All you have to do is call: :func:`datasets.load_dataset` to take your first step. This function is a true workhorse in every sense because it builds and loads every dataset you use.
+
+ELI5: ``load_dataset``
+-------------------------------
+
+Let's begin with a basic Explain Like I'm Five.
+
+At first, :func:`datasets.load_dataset` downloads and imports the dataset loading script associated with the requested dataset from the Hugging Face Hub. The Hub is a central repository where all the Hugging Face datasets and models are stored. Code in the loading script defines the dataset information (description, features, URL to the original files, etc.), and tells 🤗 Datasets how to generate and display examples from it.
+
+.. seealso::
+
+   Read the :doc:`Share <./share>` section for a step-by-step guide on how to write your own dataset loading script!
+
+The loading script downloads the dataset files from the original URL, and caches the dataset in an Arrow table on your drive. If you've downloaded the dataset before, then 🤗 Datasets will reload it from the cache to save you the trouble of downloading it again. Finally, 🤗 Datasets will generate the dataset according to the code defined in the loading script.
+
+Now that you have a high-level understanding about how datasets are built, let's take a closer look at the nuts and bolts of how all this works.
+
+Building a dataset
+------------------
+
+When you load a dataset for the first time, 🤗 Datasets takes the raw data file and builds it into a table of rows and typed columns. There are two main classes responsible for building a dataset: :class:`datasets.BuilderConfig` and :class:`datasets.DatasetBuilder`. 
+
+.. image:: /imgs/builderconfig.png
+   :align: center
+
+:class:`datasets.BuilderConfig`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`datasets.BuilderConfig` is the base class of :class:`datasets.DatasetBuilder`. The :class:`datasets.BuilderConfig` contains the following basic attributes about a dataset:
```

Comment:
I would not use "base class" here: in the Python community, this expression is used as a synonym for "parent class".

Reply:
Indeed, maybe use this ?
```suggestion
:class:`datasets.BuilderConfig` is the configuration class of :class:`datasets.DatasetBuilder`. The :class:`datasets.BuilderConfig` contains the following basic attributes about a dataset:
```